x-pack/plugins/security_solution/public/common/components/with_data_view/translations.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const DATA_VIEW_MANDATORY = i18n.translate(
+  'xpack.securitySolution.components.dataViewMandatory',
+  {
+    defaultMessage: 'DataView is mandatory but was not provided',
+  }
+);
+
+export const DATA_VIEW_MANDATORY_MSG = i18n.translate(
+  'xpack.securitySolution.components.dataViewMandatoryDetailMessage',
+  {
+    defaultMessage:
+      'It looks like DataView is not selected properly. Please select the appropriate DataView and index patterns',
```

Comment:
nit
```suggestion
      The DataView is not selected properly. Please select the appropriate DataView and index patterns',
```

> It looks like
That part felt a little off based on other messages we have in the app, but feel free to leave it or chat with docs folks 😄

Reply:
Done here : cf159a76c519125e688742d8e6fd973788f47d8d